webmacs/commands/webjump.py

```diff
@@ -259,15 +266,19 @@ def get_url(prompt):
         if len(candidates) == 1:
             webjump = WEBJUMPS[candidates[0]]
 
-    if webjump and not webjump.protocol:
+    if webjump:
         if len(args) > 1:
             # we found a webjump, now look for a single completion
-            completions = [c for c in webjump.complete_fn(args[1])
-                           if c.startswith(args[1])]
+            try:
+                completions = [c for c in webjump.complete_fn(
+                    args[1]) if c.startswith(args[1])]
+            except Exception:
+                completions = []
```

Comment:
Could you add a line like logging.exception("An error occurred during completion:") in the except clause, so the exception won't be silent?